src/FsCheck/Runner.fs

```diff
@@ -120,6 +122,42 @@ module Runner =
                     yield EndShrink result
               yield! test newSize resize rnd1 gen
         }
+        
+#if PCL // sorry windows phone no parallelism for you
+    let private parallelTest limit = test
+#else
+    let internal parallelTest limit initSize resize rnd0 gen =
+        (initSize, rnd0)
+        |> Seq.unfold (fun (initSize, rnd) ->
+            let rnd1, rnd2 = Random.split rnd
```

Comment:
Regarding benchmarks. Well... currently I've only ran tight compute-bound benchmarks, and plinq is slightly behind(!) of sequential runner with about 10-20x memory consumption(!!!). So I've implemented another parallel version using threadpool which is 1.5-2 times faster than sequential (on my 4-core laptop). But anyway I expected speedups to be somewhere close to linear. I need little more time to craft io-bound (maybe artificially via `Task.Delay`) and will provide full report then.